modules/microshift-nw-create-http-based-route.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-configuring-routes.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-nw-creating-a-route_{context}"]
+= Creating an HTTP-based route
+
+A route allows you to host your application at a public URL. It can either be secure or unsecured, depending on the network security configuration of your application. An HTTP-based route is an unsecured route that uses the basic HTTP routing protocol and exposes a service on an unsecured application port.
+
+The following procedure describes how to create a simple HTTP-based route to a web application, using the `hello-microshift` application as an example.
+
+.Prerequisites
+* You installed the OpenShift CLI (`oc`).
+* You have root access to your {microshift-short} cluster.
+* You have a web application that exposes a port and a TCP endpoint listening for traffic on the port.
+
+.Procedure
+
+. Create a service called `hello-microshift` by running the following command:
```

Comment:
The command below is creating the route, not the service. This will be a route that will take `www.example.com` as hostname. I think we may want to use `example.com` instead, and have this command in the next step.
Here you would need `oc expose pod hello-microshift -n $namespace`